### DISCUSSION TITLE
[Bug]: qBittorrent-nox not downloading on termux anymore?

### DISCUSSION THREAD

**Discussion by erickyun** [userdocs/qbittorrent-nox-static#134] [General]
### Problem description

Hi,

I Installed qBittorrent-nox on proot-distro ubuntu it worked fine before but now it won't start downloading torrents anymore?

also gived permission with chmod u+x

### Steps to reproduce the behavior.

$ uname -m
aarch64

```
wget -qO /usr/bin/qbittorrent-nox https://github.com/userdocs/qbittorrent-nox-static/releases/latest/download/aarch64-qbittorrent-nox
chmod 700 /usr/bin/qbittorrent-nox


### What is the expected behavior?

qbittorrent-nox status is always on stalled!
The status won't change to downloading anymore?

### System information

* Termux application version: 0.118.0
* Android OS version: Android 11 
* Device model: Xiaomi Redmi Note 8 Pro
```

[qbittorrent.log](https://github.com/userdocs/qbittorrent-nox-static/files/14232015/qbittorrent.log)

**Comment by userdocs** [comment]
I don't really understand the scope of the issue. It's possible it's openssl related

1: does it work with [older versions](https://github.com/userdocs/qbittorrent-nox-static/tags), recent revisions will have openssl 3.2.1 but this revision is 1 week ago so if your issue happened after this point with this build i.e. working then stopped. I might not be related.

2: if it is openssl related the hardcoded location is `--openssldir="/etc/ssl"` so make sure your env has this, even via symlink. Ubuntu should be fine but you not doing normal stuff, you're using a proot on android.

3: This repo is a build script so the issue unrelated to the script and the outcome. I'll convert this to a discussion to reflect that.

4: If it was working but now isn't you need to narrow down the issue by testing older builds to show if a build combo is the direct cause of this. Then we could look at which dependency is the problem and why.

5: You can do this by forking the repo and using the custom tag workflow if needed.
